client/landing/custom-editor/section.ts

```diff
@@ -0,0 +1,8 @@
+export const CUSTOM_EDITOR_SECTION_DEFINITION = {
+	name: 'custom-editor',
```

Comment:
In case we decide to proceed with this, I think this should be replaced with `site-editor` or similar to avoid confusion.